### PR TITLE
Init express server

### PR DESCRIPTION
**Pull request by minsulee2**
Updates
- Init express
- Add pseudo handlers
- Add error handlers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,16 @@
-# ainize-openai-assistans-service
\ No newline at end of file
+# ainize-openai-assistans-service
```
Comment (platfowner) [inline README.md:1]: how about adding a description like "... A Wrapper Ainize Container for OpenAI Assistant API..."?

[the remark this answers is not in the document]
Comment (minsulee2) [reply README.md:1]: Done

Comment (minsulee2) [reply README.md:1]: Done!

**Comment by minsulee2** [comment]
PTAL! @jiyoung-an

**Review by jiyoung-an — APPROVED** [review]
LGTM, Thanks!

File: index.js
[the remark this answers is not in the document]
Comment (jiyoung-an) [reply index.js:1]: Context sync is complete! thanks to @minsulee2 .

**Comment by minsulee2** [comment]
Thanks for the fast reviews! Merging now :)

**Pull request merged**